motion_tracker/utils/crop_vis.py

```diff
@@ -41,3 +41,18 @@ def show_stages_of_random_crop(img, box_coords, output_width=256, output_height=
                                                   output_width, output_height)
     add_box(final_img, box_coords)
     show_img("Random Crop", final_img)
+
+def show_single_stage(img, box_coords, title="Happy Dance Image"): 
```

Comment:
Cool. I thought the same thing, and had started adjusted the function directly above that but wanted to hold off.